src/org/traccar/reports/ReportUtils.java

```diff
@@ -35,4 +39,45 @@ private ReportUtils() {
         return result;
     }
 
+    public static double calculateDistance(Position firstPosition, Position lastPosition) {
+        return calculateDistance(firstPosition, lastPosition, true);
+    }
+
+    public static double calculateDistance(Position firstPosition, Position lastPosition, boolean useOdometer) {
+        double distance = 0.0;
+        if (useOdometer && firstPosition.getAttributes().containsKey(Position.KEY_ODOMETER)
+                && lastPosition.getAttributes().containsKey(Position.KEY_ODOMETER)) {
+            distance = (((Number) lastPosition.getAttributes().get(Position.KEY_ODOMETER)).doubleValue()
+                    - ((Number) firstPosition.getAttributes().get(Position.KEY_ODOMETER)).doubleValue())
+                    * 1000;
```

Comment:
Agree, It will be more uniform.

I was wondering and actually I'm not sure that odometer is always in kilometers. What do you think can it be in miles?